docs/guides/security/basics/how-to-configure-automated-security-updates-debian/index.md

```diff
@@ -0,0 +1,170 @@
+---
+slug: how-to-configure-automated-security-updates-debian
+author:
+  name: Hackersploit
+description: 'Keeping your system up-to-date with the latest packages and security updates can be a tedious task. Automate security (and other package) updates with the utility Unattended Upgrades on Debian.'
+og_description: 'Keeping your system up-to-date with the latest packages and security updates can be a tedious task. Automate security (and other package) updates with the utility Unattended Upgrades on Debian.'
+keywords: ["debian unattended-upgrades", "configuring unattended-upgrades"]
+tags: ["debian", "security"]
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+modified: 2020-10-22
+modified_by:
+  name: Linode
+published: 2020-10-22
+title: How to Configure Automated Security Updates on Debian
+h1_title: Configuring Automated Security Updates on Debian
+aliases: ['/security/basics/how-to-configure-automated-security-updates-debian/']
+relations:
+    platform:
+        key: automated-security-upgrades
+        keywords:
+            - distribution: Debian
+image: Configuring_automated_security_updates_debian.png
+---
+
+Keeping your system up-to-date with the latest packages and security updates can be a tedious task. Most users forget to do it, leaving them vulnerable to countless threats. Automate security (and other package) updates with the utility [Unattended Upgrades](https://wiki.debian.org/UnattendedUpgrades) on Debian.
+
+## Before You Begin
+
+1.  Complete the [Getting Started](/docs/guides/getting-started) guide.
+
+1.  Follow the [Securing Your Server](/docs/guides/securing-your-server/) guide to create a standard user account, and harden SSH access.
+
+1.  Log into your Linode via SSH and update and upgrade.
+
+        sudo apt update && sudo apt upgrade
+
+{{< note >}}
+This guide is written for a non-root user. Commands that require elevated privileges are prefixed with `sudo`. If you’re not familiar with the `sudo` command, see our [Users and Groups](/docs/guides/linux-users-and-groups) guide.
+{{< /note >}}
+
+## Install Unattended Upgrades
+
+You can set up automated security updates on Debian or Ubuntu by installing a helpful utility called `unattended-upgrades`.
```

Comment:
Can probably remove Ubuntu from this line

Reply:
done